public/partials/inventory/data-inventory.html

```diff
@@ -35,11 +35,107 @@ <h1 class="md-toolbar-tools">Issues and Notes</h1>
       <md-content layout-padding>
 
         <md-list class="fixedRows">
-          <md-list-item class="md-2-line contact-item" ng-repeat="(key, value) in selectedRow.entity">
+
```

Comment:
We have to indicate to the user what was present in the spreadsheets...  The values I had been displaying before was the values actually being imported that failed to match.  We can't remove these values unless there was a match for each of the objects.